lib/modules/platform/gitlab/readme.md

```diff
@@ -2,31 +2,29 @@
 
 ## Authentication
 
-First, [create a Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) for the bot account.
+First, [create a Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) for the bot account, or use a [Group Access Token](https://docs.gitlab.com/ee/user/group/settings/group_access_tokens.html#bot-users-for-groups).
 
 If you are using a Group access token, the token must have Developer role or higher permissions in order to create issues and merge requests.
 The token must have Maintainer permissions in order to perform Automerge.
+Ensure that you are [rotating the group access token](https://docs.gitlab.com/ee/api/group_access_tokens.html#rotate-a-group-access-token) before it expires in order to keep the same GitLab-generated bot user.
 
-For real runs, give the PAT these scopes:
+For real runs, give the access token these scopes:
```

Comment:
Do personal access and group access tokens need the same permissions? It's not immediately obvious if the instructions/rules are the same for both PATs and GATs to me.

If the permissions/rules are different for PATs, than for GATs, we should have two sections:

- one section for PAT
- one section for GAT

But if both tokens have the same instructions/rules, we need a introductory sentence that explains that we will call both of these "access tokens" in the following text.

Reply:
@HonkingGoose Thanks for the suggestions above and the detailed explanations on why 👍 

If we are ok with removing `read_user` from the list (as it is not required, see above), then the permissions are the same. I changed it to "access token" because of that. It's a good point that an introductory sentence should explain this. I added one and replaced two other occurrences of "PAT" later in the instructions.